resources/assets/lib/models/chat/message.ts

```diff
@@ -9,6 +9,8 @@ import * as moment from 'moment';
 import core from 'osu-core-singleton';
 import { linkify } from 'utils/url';
 
+export const maxLength = 1024;
```

Comment:
as it turned out there are two different max length on the backend; one is for announcement (no limit) and the other is limited to `config('osu.chat.message_length_limit')`:

https://github.com/ppy/osu-web/blob/master/app/Models/Chat/Channel.php#L400

Reply:
probably should also update the check there to account for column size. Something like `$lengthLimit = $this->isAnnouncement() ? 1024 : config(...)`.